.pgpass

```diff
@@ -0,0 +1 @@
+localhost:*:*:postgres:postgres
```

Comment:
why is this needed?

Reply:
in the ci config we must set a POSTGRESS_PASSWORD value, and when we run the psql [command](https://github.com/fastify/fastify-postgres/pull/60/files#diff-60fe268f71ae7f8409aff40d0a3e51d8R37) it would prompt for a password, but if we provide a `.pgpass` file it will use that to get the password